packages/aws-cdk-lib/cx-api/FEATURE_FLAGS.md

```diff
@@ -1298,4 +1300,22 @@ When this featuer flag is enabled, the default volume type of the EBS volume wil
 **Compatibility with old behavior:** Pass `volumeType: EbsDeviceVolumeType.GENERAL_PURPOSE_SSD` to `Volume` construct to restore the previous behavior.
 
 
+### @aws-cdk/pipelines:reduceAssetRoleTrustScope
+
+*Remove the root account principal from PipelineAssetsFileRole trust policy* (default)
+
+Remove the root account principal from the generated PipelineAssetsFileRole trust policy,
+because it's only assumed by the codebuild.
+Use a feature flag to make sure existing customers who might be relying
+on the overly-broad trust policy are not broken.
```

Comment:
Instead of explaining why we add a feature flag here, can we explain the behaviour of the flag when it's true vs false, i.e. `When this feature flag is enabled, the it will not add root account principal ...`